assignments/objects.js

```diff
@@ -19,26 +19,84 @@ const example = {
 
 // Write your intern objects here:
 
+const interns = [
+  {"id": 1 , "name": "Mitzi", "email":"mmelloy0@psu.edu", "gender": "F"},
+  {"id": 2, "name": "Kennan", "email":"kdiben1@tinypic.com", "gender": "M"},
+  {"id": 3, "name": "Keven", "email":"kmummery2@wikimedia.org", "gender": "M"},
+  {"id": 4, "name": "Gannie", "email":"gmartinson3@illinois.edu", "gender": "M"},
+  {"id": 5, "name": "Antonietta", "email":"adaine5@samsung.com", "gender": "F"}
+]
+
+ const mitzi = interns[0];
+const kennan = interns[1];
+const keven = interns[2];
+const gannie = interns[3];
+const antonietta = interns[4];
+
+
+// const Mitzi = {
+//   "id": 0,
+//   "name": "Mitzi",
+//   "email": "mmelloy0@psu.edu",
+//   "gender": "F",
+// }
+
+// const Kennan = {
+//   "id": 1,
+//   "name": "Kennan",
+//   "email": "kdiben1@tinypic.com",
+//   "gender": "M",
```

Comment:
```suggestion
//   "gender": "M",
speak: function() {
    return "Hello, my name is Kennan!";
  }
```

Reply:
add this the function on the object and uncomment the console.log below to get it rendering. Try something similar on the Antonietta fn.